code/modules/mining/equipment_locker.dm

```diff
@@ -430,7 +430,7 @@
 		new /datum/data/mining_equipment("Silver Pickaxe",				/obj/item/pickaxe/silver,											1000),
 		new /datum/data/mining_equipment("Mining Conscription Kit",		/obj/item/storage/backpack/duffel/mining_conscript,					1500),
 		new /datum/data/mining_equipment("Jetpack",             		/obj/item/tank/jetpack/carbondioxide/mining,               			2000),
-		new /datum/data/mining_equipment("Mining Hardsuit",				/obj/item/storage/box/mininghardsuit,								2000),
+		new /datum/data/mining_equipment("Mining Hardsuit",				/obj/item/clothing/head/helmet/space/hardsuit/mining,				2000),
```

Comment:
Shouldnt this give you the hardsuit? Unless im misunderstanding this the vendor will give you the helmet alone right now.

Reply:
oh, I made it give the wrong thing. Lol.

Reply:
:+1: